sql/pgsql/_debug_api.sql

```
@@ -12,6 +12,15 @@ update public."MatchupPreview" set
 "PredictedStraightUpWinner" = 'f3c03a13-7806-c144-9e0e-0d4910ac770d',
 "PredictedSpreadWinner" = 'f3c03a13-7806-c144-9e0e-0d4910ac770d' WHERE "Id" = '2b4b3533-1b75-4e41-a4a5-1d62c0996545'
 */
+select * from public."Prompt";
+
+select * from public."MatchupPreview" where "PromptId" is null;
```

SQLFluff (4.3.0)

[error] The 'where' keyword should always start a new line.

(LT14)